examples/example-app/src/main/scala/org/virtuslab/example/CirceAkkaSerializable.scala

```diff
@@ -0,0 +1,6 @@
+package org.virtuslab.example
+
+import org.virtuslab.ash.annotation.SerializabilityTrait
+
+@SerializabilityTrait
+trait CirceAkkaSerializable extends Product with Serializable
```

Comment:
Pls update CI so that:
1. this particular example project is being
2. both example project are scalafmt-checked just as the top-level project

Reply:
> 1. this particular example project is being

What do you mean? I don't understand this point.

Reply:
Sorry, updated now... build this project in CI, just as event-migration is built